### PR TITLE
Adds venv install steps and package requirements list

### PR DESCRIPTION
**Pull request by gannonprudhomme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
+2) To install the packages, you first need to make a virtual environment for Python, which will help us ensure that our libraries & Python versions are unified. You can do so be by running: 
+    - `python3 -m venv env`
+        - This creates a virtual environment in the `env/` folder. 
+        - In here you'll see a directory containing the libraries that will be installed in the next step(`Lib/` on Windows)
+        - You'll also see a folder(`Scripts/` on Windows & `bin/` on Unix systems) that contains the scripts that you need to start the virtual environment.
+    - Next, for Unix systems run: `source env/bin/activate`
+    - And for Windows, in Powershell run: `./env/Scripts/activate`
+        - If you get an permisions error that says: `execution of scripts is disabled on this system`, then open another PowerShell as Administrator and run `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned`.
+            - Alternatively, you can just run the original script as Administrator.
+    - You will need to do `source env/bin/activate` or `./env/Scripts/activate` anytime you want to interact with the server(i.e. to run it with `python manage.py runserver`)
```
Comment (gannonprudhomme) [inline README.md]: Not positive about this but I assumed this was the case/is true

Comment (gannonprudhomme) [reply README.md]: Also need to add the `pip install -r requirements.txt`

File: README.md [hunk 2]
```diff
@@ -2,11 +2,16 @@
...
-pip install psycopg2
-```
+2) To install the packages, you first need to make a virtual environment for Python, which will help us ensure that our libraries & Python versions are unified. You can do so be by running: 
+    - `python3 -m venv env`
+        - This creates a virtual environment in the `env/` folder. 
+        - In here you'll see a directory containing the libraries that will be installed in the next step(`Lib/` on Windows)
+        - You'll also see a folder(`Scripts/` on Windows & `bin/` on Unix systems) that contains the scripts that you need to start the virtual environment.
+    - Next, for Unix systems run: `source env/bin/activate`
+    - And for Windows, in Powershell run: `./env/Scripts/activate`
+        - If you get an permisions error that says: `execution of scripts is disabled on this system`, then open another PowerShell as Administrator and run `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned`.
```
Comment (LucasArnold) [inline README.md]: If you get a permission error*

Comment (gannonprudhomme) [reply README.md]:
```suggestion
        - If you get an permissions error that says: `.
```

**Pull request merged**